### ISSUE TITLE
A `ModuleNotFoundError` is reported when import cobra

### ISSUE THREAD

**Issue by JHL-452b** [leilahsn/FastKnock#2]
Hello, 

There is a `ModuleNotFoundError` when I import `cobra.test` in **FastKnock.py**. 
I noticed that Cobrapy stopped distributed tests after [version 0.25](https://github.com/opencobra/cobrapy/blob/devel/release-notes/0.25.0.md), but my cobrapy is 0.27, which is why this error occurs. This may not be consistent with what you mentioned in the README that requires cobra versions >=  0.21.

**Comment by 2727-bit** [comment]
Yes, I also encountered this issue. Have you solved your problem?
